spec/dor/services/client/object_spec.rb

```diff
@@ -120,10 +120,13 @@
 
   describe '#publish' do
     subject(:request) { client.publish(workflow: 'accessionWF') }
+    subject(:no_wf_request) { client.publish }
```

Comment:
I think it's easier to read if you keep each subject in it's own context.  one that is "with workflow" and one "without workflow"

Reply:
I can clean that up, I wasn't sure in this test as it was already context heavy, but can review